pyartcd/pyartcd/pipelines/operator_sdk_sync.py

```diff
@@ -0,0 +1,133 @@
+        self.assembly = assembly
+        self.updatelatest = updatelatest
+        self.sdk = "operator-sdk"
+
+        self._jira_client = runtime.new_jira_client()
+        ocp_build_data_repo = runtime.new_github_client().get_repo("openshift/ocp-build-data")
+        release_file = ocp_build_data_repo.get_contents("releases.yml", ref=group)
+        release_yaml = yaml.load(release_file.decoded_content, Loader=yaml.FullLoader)
+        self.extra_ad_id, self.parent_jira_key = self.get_ad_jira_key(assembly, release_yaml)
+
+    def run(self):
+        advisory = Erratum(errata_id=self.extra_ad_id)
+        self._logger.info("Check advisory status ...")
+        if advisory.errata_state in ["QE", "NEW_FILES"]:
+            self._logger.info("Advisory status not in REL_PREP yet ...")
+            return
+        if advisory.errata_state == "SHIPPED_LIVE":
+            self._logger.info("Advisory status already in SHIPPED_LIVE, update subtask 9 ...")
+            self._update_jira(self.parent_jira_key, 8, "Advisory status already in SHIPPED_LIVE")
+        self._logger.info("Advisory status already in post REL_PREP, update subtask 7 ...")
+        self._update_jira(self.parent_jira_key, 6, "Advisory status already in REL_PREP")
+
+        et_builds = advisory.errata_builds
+        sdk_build = [i for i in et_builds[f'OSE-{self.version}-RHEL-8']
+                     if re.search("openshift-enterprise-operator-sdk-container*", i)]
+        if not sdk_build:
+            self._logger.info("No SDK build to ship, update subtask 8 then close ...")
+            self._update_jira(self.parent_jira_key, 7,
+                              f"No SDK build to ship, operator_sdk_sync job: {os.environ.get('BUILD_URL')}")
+            return
+
+        build = koji.ClientSession(constants.BREW_SERVER).getBuild(sdk_build[0])
+        archlist = ["amd64", "arm64", "ppc64le", "s390x"]
+        sdkVersion = self._get_sdkversion(build['extra']['image']['index']['pull'][0])
+        self._logger.info(sdkVersion)
+        for arch in archlist:
+            self._extract_binaries(arch, sdkVersion, build['extra']['image']['index']['pull'][0])
+        self._update_jira(self.parent_jira_key, 7,
+                          f"operator_sdk_sync job: {os.environ.get('BUILD_URL')}")
+
+    def get_ad_jira_key(self, assembly, release_yaml):
+        if 'group' in release_yaml['releases'][assembly]['assembly'].keys():
+            extra_ad_id = release_yaml['releases'][assembly]['assembly']['group']['advisories']['extras']
+            parent_jira_key = release_yaml['releases'][assembly]['assembly']['group']['release_jira']
+            return extra_ad_id, parent_jira_key
+        if 'assembly' not in release_yaml['releases'][assembly]['assembly']['basis'].keys():
+            raise ValueError("Can not find jira and advisory number from assembly")
+        return self.get_ad_jira_key(release_yaml['releases'][assembly]['assembly']['basis']['assembly'], release_yaml)
```

Comment:
You can use `doozer config:read-group` or `load_group_config` in util.py.

Reply:
I think load_group_config gets group.yml while we need extra advisory id and parent jira id

Reply:
No. `load_group_config` calls `doozer config:read-group` internally, which merges overrides in releases.yml.